### PR TITLE
fit the emoji panel to message tag

### PR DESCRIPTION
**Pull request by VirajDilshanKumarage**
fit the emoji panel to message tag

1jitsi
2jitsi

if any issue feel free to contact me via email: viraj123e@gmail.com

### CODE REVIEW

**Comment by VirajDilshanKumarage** [comment]
please review this code

File: react/features/chat/components/web/ChatInput.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import data from '@emoji-mart/data';
```
Comment (saghul) [inline react/features/chat/components/web/ChatInput.tsx:1]: Can you please import this as "emojiData", so it's not so generic?

File: react/features/chat/components/web/ChatInput.tsx [hunk 2]
```diff
@@ -64,9 +65,11 @@ class ChatInput extends Component<IProps, IState> {
 
     state = {
         message: '',
+        messages: [],
```
Comment (saghul) [inline react/features/chat/components/web/ChatInput.tsx:68]: Unused.

File: react/features/chat/components/web/ChatInput.tsx [hunk 3]
```diff
@@ -64,9 +65,11 @@ class ChatInput extends Component<IProps, IState> {
...
         showSmileysPanel: false
     };
 
+
```
Comment (saghul) [inline react/features/chat/components/web/ChatInput.tsx:72]: Please avoid newline changes.

File: react/features/chat/components/web/ChatInput.tsx [hunk 4]
```diff
@@ -123,11 +128,16 @@ class ChatInput extends Component<IProps, IState> {
                 <div id = 'chat-input' >
                     {!this.props._areSmileysDisabled && this.state.showSmileysPanel && (
                         <div
-                            className = 'smiley-input'>
+                            className = 'smiley-input' >
                             <div
                                 className = 'smileys-panel' >
-                                <SmileysPanel
```
Comment (saghul) [inline react/features/chat/components/web/ChatInput.tsx:129]: Is this smileys panel used anywhere else now? If not we should delete it.

**Review by saghul — CHANGES_REQUESTED** [review]

File: react/features/chat/components/web/ChatInput.tsx [hunk 5]
```diff
@@ -147,7 +157,9 @@ class ChatInput extends Component<IProps, IState> {
                         accessibilityLabel = { this.props.t('chat.sendButton') }
                         disabled = { !this.state.message.trim() }
                         icon = { IconSend }
-                        onClick = { this._onSubmitMessage }
+                        onClick={() => {
+                            this._onSubmitMessage() 
+                            this._toggleSmileysPanel();}}
```
Comment (shawnchin) [inline react/features/chat/components/web/ChatInput.tsx:162]: Is this correct? Seems strange to toggle on/off the smiley panel each time a message is sent.

**Pull request closed without merging**